app/components/global/AppBar/AppBar.js

```diff
@@ -42,16 +42,15 @@ const Separator = styled.div`
   margin-right: ${th('space.2')};
 `
 const DesktopView = styled(Flex)`
-  width:80%
   align-items: center;
   display: none;
-  ${media.mobileUp`
+  ${media.tabletPortraitUp`
     display: flex;
   `};
 `
 
 const AppBar = ({ user, menuItems }) => (
```

Comment:
Might be good to add a destructuring assignment on `user` that matches your defaultProps below?